docs/uglocalpv_volume_encrypt.md

```diff
@@ -0,0 +1,197 @@
+```
+sudo cryptsetup luksFormat /dev/<device_name>
+```
+Example command:
+```
+sudo cryptsetup luksFormat /dev/nvme2n1p1
+```
+Example output:
+```
+WARNING!
+========
+This will overwrite data on /dev/nvme2n1p1 irrevocably.
+
+Are you sure? (Type uppercase yes): YES
+Enter passphrase:
+Verify passphrase:
+```
+
+### Step 4: Create a LUKS device mapping
+```
+sudo cryptsetup luksOpen /dev/<device_name> <mapping>
+```
+Example command:
+```
+sudo cryptsetup luksOpen /dev/nvme2n1p1 backup1
+Enter passphrase for /dev/nvme2n1p1:
+```
+
+Confirm the new LUKS partition by running `lsblk` command. In our example, mapped as `backup1`.
+
+Example output:
+```
+NAME        MAJ:MIN RM    SIZE RO TYPE  MOUNTPOINT
+nvme1n1     259:0    0   69.9G  0 disk
+nvme0n1     259:1    0    128G  0 disk
+├─nvme0n1p1 259:2    0 1007.5K  0 part
+└─nvme0n1p2 259:3    0    128G  0 part  /
+nvme2n1     259:4    0    100G  0 disk
+└─nvme2n1p1 259:6    0     20G  0 part
+  └─backup1 254:0    0     20G  0 crypt
+```
+
+### Step 5: Verify status of LUKS partition
+```
+sudo cryptsetup -v status <mapping>
+```
+Example command:
+```
+sudo cryptsetup -v status backup1
+```
+Example output:
+```
+/dev/mapper/backup1 is active.
+  type:    LUKS1
+  cipher:  aes-xts-plain64
+  keysize: 256 bits
+  device:  /dev/nvme2n1p1
+  offset:  4096 sectors
+  size:    41938945 sectors
+  mode:    read/write
+Command successful.
+admin@ip-172-20-44-175:~$ sudo mkfs.ext4 /dev/mapper/backup1
+mke2fs 1.43.4 (31-Jan-2017)
+Creating filesystem with 5242368 4k blocks and 1310720 inodes
+Filesystem UUID: 53e21785-574d-44f4-a916-7706a54b28b5
+Superblock backups stored on blocks:
+        32768, 98304, 163840, 229376, 294912, 819200, 884736, 1605632, 2654208,
+        4096000
+
+Allocating group tables: done
+Writing inode tables: done
+Creating journal (32768 blocks): done
+Writing superblocks and filesystem accounting information:
+done
+```
+
+### Step 6: Mount LUKS partition for Local PV Hostpath use
+```
+sudo mkdir -p <localpv_mount_point>
+sudo mount <mapping> <localpv_mount_point>
+```
+Example command:
+```
+sudo mkdir -p /var/openebs/local
+sudo mount /dev/mapper/backup1 /var/openebs/local
+```
+
+Confirm the mount point by running `lsblk` command. In our example, mounted as as `var/openebs/local`.
+
+Example output:
+```
+NAME        MAJ:MIN RM    SIZE RO TYPE  MOUNTPOINT
+nvme1n1     259:0    0   69.9G  0 disk
+nvme0n1     259:1    0    128G  0 disk
+├─nvme0n1p1 259:2    0 1007.5K  0 part
+└─nvme0n1p2 259:3    0    128G  0 part  /
+nvme2n1     259:4    0    100G  0 disk
+└─nvme2n1p1 259:6    0     20G  0 part
+  └─backup1 254:0    0     20G  0 crypt /var/openebs/local
+```
+
+### Step 7: Create an example pod using Local PV Hostpath volume
+```
+kubectl apply -f https://openebs.github.io/charts/examples/local-hostpath/local-hostpath-pvc.yaml
+kubectl apply -f https://openebs.github.io/charts/examples/local-hostpath/local-hostpath-pod.yaml
```

Comment:
Also, should we mention that in there in only single device available, ensure to add nodeSelector in the application spec?